client/src/components/GameReviewForm.jsx

```diff
@@ -3,11 +3,21 @@
 
 import { Button } from "primereact/button";
 import { InputNumber } from "primereact/inputnumber";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import axios from "axios";
+import { Rating } from "primereact/rating";
 
 function GameReviewForm({ setGameReviewFormVisible, gameDetails, loggedInUser, setNewReviewSubmitted, baseURL }) {
     
+    const [reviewFormSubmitMessage, setReviewFormSubmitMessage] = useState("");
+    
+    // Timer for error message
```

Comment:
what is the timer doing here?

Reply:
This timer causes the error message to disappear on the screen after it's been displayed for 5 seconds